airbyte-ci/connectors/live-tests/src/live_tests/commons/connector_runner.py

```diff
@@ -134,7 +149,8 @@ async def _run(
                 [
                     "sh",
                     "-c",
-                    " ".join(airbyte_command) + f" > {self.IN_CONTAINER_OUTPUT_PATH} 2>&1 | tee -a {self.IN_CONTAINER_OUTPUT_PATH}",
+                    " ".join(airbyte_command)
+                    + f"| {self.IN_CONTAINER_OBFUSCATOR_PATH} > {self.IN_CONTAINER_OUTPUT_PATH} 2>&1 | tee -a {self.IN_CONTAINER_OUTPUT_PATH}",
```

Comment:
I'm realizing that the way we're piping stuff out is not great: we're mixing stdout and stderr which means that we're altering the original connector logic.
I think we should pipe stdout and stderr separatly to the obfuscator and the obfuscator would tee to the correct output.

```bash
airbyte_command > >(obfuscator stdout) 2> >(obfuscator stderr)
```

(it would also require changing the obfuscator entrypoint to make it accept a `write_to` parameter which would be `stderr` or `stdout`.